docs/source/usage/modes/thread.rst

```diff
@@ -40,7 +40,7 @@ The following commands are available in thread mode
     move focus in current buffer
 
     argument
-        up, down, page up, page down, first, last
+        up, down, [half]page up, [half]page down, first, last,                   parent, first reply, last reply,                   next sibling, previous sibling, next, previous,                   next unfolded, previous unfolded,                   next NOTMUCH_QUERY, previous NOTMUCH_QUERY
```

Comment:
This is autogenerated from the docstring. I tried to format the docstring and did not care about this. This whitespace will become irellevant in the html output.